integration/ipam/ipam.go

```diff
@@ -37,7 +37,7 @@ func GenerateFlannelNetwork(ctx context.Context, clusterID string, crdStorage mi
 
 	cidrMask := net.CIDRMask(flannelCidrSize, 32)
 
-	cidr, err := ipamService.CreateSubnet(ctx, cidrMask, flannelNetworkAnnotation(clusterID))
+	cidr, err := ipamService.CreateSubnet(ctx, cidrMask, flannelNetworkAnnotation(clusterID), nil)
```

Comment:
The service got changed. Only adapting. I think the reservation could be reworked and improved but not doing this now.